src/interfaces/new-request-interface.ts

```diff
@@ -67,6 +67,7 @@ export interface NewRequestIF {
     request: RequestActionsI
   }
   showActualInput: boolean
+  societiesModalVisible: boolean
```

Comment:
I had suggested just passing a prop to the dialog to show/hide it as that would be cleanest.

It's OK that you figured out how the other modals in Namerequest UI are show/hidden.